packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -179,13 +176,17 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     uniqueInvitations,
   });
 
+  const inviter = { name: inviterName };
+
   if (invitationsForNewUsers.length) {
     await handleNewUsersInvites({
       invitationsForNewUsers,
       team,
       orgConnectInfoByUsernameOrEmail,
-      input,
-      inviter: ctx.user,
+      teamId: team.id,
+      language,
+      isOrg: isTeamAnOrg,
```

Comment:
This is where we send the `isOrg` value directly from server instead of relying on request param